servers/src/test/java/tachyon/master/block/BlockMasterTest.java

```diff
@@ -220,6 +227,51 @@ public void unknownHeartbeatTest() throws Exception {
     Assert.assertEquals(new Command(CommandType.Register, ImmutableList.<Long>of()), heartBeat);
   }
 
+  @Test
+  public void detectLostWorkerTest() throws Exception {
+    HeartbeatScheduler.await(HeartbeatContext.MASTER_LOST_WORKER_DETECTION, 5, TimeUnit.SECONDS);
+
+    // Get a new worker id.
+    long workerId = mMaster.getWorkerId(new NetAddress("localhost", 80, 81));
+    MasterWorkerInfo workerInfo = mPrivateAccess.getWorkerById(workerId);
+    Assert.assertNotNull(workerInfo);
+
+    // Run the lost worker detector. No workers will be considered lost.
```

Comment:
The `No workers ...` part of the comment is repeated below and seem unnecessary here.